patches/trezor-connect+8.1.30-beta.7.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/trezor-connect/lib/typescript/networks/bitcoin.d.ts b/node_modules/trezor-connect/lib/typescript/networks/bitcoin.d.ts
+index 0c4ad87..66b0ffb 100644
+--- a/node_modules/trezor-connect/lib/typescript/networks/bitcoin.d.ts
++++ b/node_modules/trezor-connect/lib/typescript/networks/bitcoin.d.ts
```

Comment:
rebase over develop should help, this patch is no longer necessary

Reply:
https://github.com/trezor/trezor-suite/pull/4297/commits/9c16607512af09a5f8125a6fd7ccb7fb72e15127